src/core/calibration.py

```
@@ -0,0 +1,254 @@
+"""Calibration functionality for capture region setup."""
+
+import mss
+from pathlib import Path
+from typing import Optional, Dict
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import: `typing.Dict`

(F401)